go/vt/vtgate/evalengine/compiler_fn.go

```diff
@@ -958,3 +960,45 @@ func (c *compiler) compileFn_DateFormat(call *builtinDateFormat) (ctype, error)
 	c.asm.jumpDestination(skip1, skip2)
 	return ctype{Type: sqltypes.VarChar, Col: col, Flag: arg.Flag | flagNullable}, nil
 }
+
+func (c *compiler) compileFn_ConvertTz(call *builtinConvertTz) (ctype, error) {
+	n, err := c.compileExpr(call.callable()[0])
+	if err != nil {
+		return ctype{}, err
+	}
+
+	from, err := c.compileExpr(call.callable()[1])
+	if err != nil {
+		return ctype{}, err
+	}
+
+	to, err := c.compileExpr(call.callable()[2])
+	if err != nil {
+		return ctype{}, err
+	}
+
+	skip1 := c.compileNullCheck3(n, from, to)
+	var skip2 *jump
+
+	switch {
```

Comment:
any reason you are doing `switch` instead of `if` here?

Reply:
Guess mostly consistent style with the rest.